docs/client/rest/rest_api.md

```diff
@@ -349,6 +349,43 @@ Gets the local log lines from this batch.
 | logRowSet | The log lines     | List of sting |
 | rowCount  | The log row count | Int           |
 
+## Admin Resource
+
+### POST /admin/refresh/hadoop_conf
+
+Refresh the Kyuubi server hadoop conf.
+
+### DELETE /admin/engine
+
+Delete specified engine.
```

Comment:
Delete the specified engine.

BTW, can we accurately delete the specified engine by the following parameters?

Reply:
yes, delete operation will delete an engine at a time. When the parameter is not specified, we will read the value from Kyuubi conf. For subdomain, if it is empty, we will use "default". Then we could construct the zk path.